lib/asserts.nix

```diff
@@ -3,19 +3,19 @@
 rec {
 
   /* Print a trace message if pred is false.
-     Intended to be used to augment asserts with helpful error messages.
+    Intended to be used to augment asserts with helpful error messages.
 
-     Example:
-       assertMsg false "nope"
-       => false
-       stderr> trace: nope
+    Example:
```

Comment:
Indentation of comments get lost. nixpkgs-fmt probably should ignore comments.

Reply:
How much work would it be to patch out the comment handling?

Reply:
> > I like the general idea of doing this.
> > However, there are some things I don't like about the current formatting:
> > 
> > * I don't think comments should be reformatted; they sometimes contain nix expressions, which are now flattened by the formatter (or it should be more intelligent, but I think not formatting is better)
> > * I saw something where multiple values were listed like below, and the formatting got weird
> > 
> > packageA,
> > # comment
> > packageB,
> > # comment
> > It was formatted to this:
> > packageA
> > , # comment
> > packageB
> > , # comment
> > The comments are really not great, the second thing is less of an issue, but I'd never manually format it like that.
> > (This was just with a brief look, haven't looked very long at this)
> 
> That looks like [nix-community/nixpkgs-fmt#243](https://github.com/nix-community/nixpkgs-fmt/issues/243)

That's fixed in nixpkgs-fmt master. Going to see if I can use that version and test it here.